akka-bbb-apps/src/main/scala/org/bigbluebutton/core/models/Polls.scala

```diff
@@ -455,6 +458,11 @@ class Poll(val id: String, val questions: Array[Question], val numRespondents: I
   private var _stopped: Boolean = false
   private var _showResult: Boolean = false
   private var _numResponders: Int = 0
+  var _responders = new ArrayBuffer[Responder]()
+
```

Comment:
Please change to private var and provide accessors instead.

Reply:
updated in https://github.com/bigbluebutton/bigbluebutton/pull/10742/commits/50f85ff2e24d287359c15d5ff5dcc7d064678c73